nixos/modules/services/hardware/pcscd.nix

```
@@ -44,6 +44,12 @@ in
         See {manpage}`reader.conf(5)` for valid options.
       '';
     };
+
+    extraArgs = mkOption {
+      type = types.str;
+      default = "";
```

```suggestion
      type = types.listOf types.str;
      default = [ ];
```

this is more extensible and can be set across different modules, etc.